felix/fv/wireguard_test.go

```diff
@@ -1775,6 +1805,9 @@ func wireguardTopologyOptions(routeSource string, ipipEnabled, wireguardIPv4Enab
 	if wireguardIPv6Enabled {
 		felixConfig.Spec.WireguardEnabledV6 = &enabled
 	}
+	if wireguardThreadingEnabled {
+		felixConfig.Spec.WireguardThreadingEnabled = &enabled
+	}
```

Comment:
The FV tests failed. I believe it's because of this part. You need to also disable the config when the option is disabled. This allows to switch back and forth in the test cases.

Reply:
Ah ok in that case this can be simplified to `felixConfig.Spec.WireguardThreadingEnabled = &wireguardThreadingEnabled`